### ISSUE TITLE
Allow the route overview FloatingButton to be removed

### ISSUE THREAD

**Issue by matthewkertis** [mapbox/mapbox-navigation-ios#2102]
We would like the ability to hide or remove the overview FloatingButton. We would also like to be able to programmatically call that action ourselves from a custom button we are adding to the bottom banner.

**Comment by frederoni** [comment]
It's not the most ergonomic, but;
```swift
let bottomBanner = navigationViewController.view.findSubviewsRecursively(BottomBannerView.self).first!
let floatingButtons = navigationViewController.view.findSubviewsRecursively(FloatingButton.self)
let overviewButton = floatingButtons[0]
let muteButton = floatingButtons[1]
let feedbackButton = floatingButtons[2]
```
would give it access to the floating buttons.
From here you could add/remove your own targets for each button. It might not be entirely safe because we find the by index here, which is subject to change. 🤔

**Comment by matthewkertis** [comment]
Yeah we considered that but we were hoping to get the buttons exposed because, as you say, they are subject to change.
